TAF_EasyRest/TestFramework/PageComponents/Owner/CreateMenuPageComponents/CreateListMenuAndComponent.cs

```diff
@@ -0,0 +1,30 @@
+﻿
+namespace TestFramework.PageComponents.Owner.CreateMenuPageComponents
+{
+    public class CreateListMenuAndComponent
+    {
+            IWebDriver driver { get; }
+
+            public CreateListMenuAndComponent(IWebDriver driver)
+            {
+                this.driver = driver;
+            }
+
+            private IWebElement _finishButton => driver.FindElement(By.XPath("//p[contains(text(), 'Try out')]/parent::div/following-sibling::div[2]"));
+            private IWebElement _backButton => driver.FindElement(By.XPath("//p[contains(text(), 'Try out')]/parent::div/following-sibling::div[1]"));
```

Comment:
I'm afraid you won't find these buttons until you make 2 steps before this.
![image](https://user-images.githubusercontent.com/14205615/183022694-3790047f-f37a-485a-8748-8db1a9b6e989.png)

Reply:
Back and Finish buttons are visible only on the third step.

Reply:
so I moved them to another component

Reply:
this component appears when we click "next" on the previously page